server/errorstore/errors.go

```diff
@@ -55,12 +57,6 @@ func NewHandler(ctx context.Context, pool fleet.RedisPool, logger kitlog.Logger,
 		runHandler(ctx, eh)
 	}
 
-	// Clear out any records that exist.
-	// Temporary mitigation for #3065.
-	if _, err := eh.Flush(); err != nil {
-		level.Error(eh.logger).Log("err", err, "msg", "failed to flush redis errors")
-	}
-
```

Comment:
heads-up @zwass @mna I'm removing this because as far as I understand the issue has been solved and this is no longer necessary.